packages/core/src/actions/types.ts

```diff
@@ -6,16 +6,6 @@ import { SphinxPreview } from '../preview'
 import { SphinxJsonRpcProvider } from '../provider'
 import { ExecutionMode } from '../constants'
 
-/**
- * Possible action types.
- */
-export const SphinxActionType = {
-  SET_STORAGE: BigInt(0),
-  DEPLOY_CONTRACT: BigInt(1),
-  CALL: BigInt(2),
-  CREATE: BigInt(3),
-}
-
```

Comment:
Super old field that was still existed in the `ActionInput` TypeScript types. Removed it because it's not necessary anymore